openupgrade_scripts/scripts/account_edi_proxy_client/14.0.1.0/upgrade_analysis.txt

```diff
@@ -11,5 +11,3 @@ account_edi_proxy_client / account_edi_proxy_client.user / refresh_token (char)
 account_edi_proxy_client / res.company              / account_edi_proxy_client_ids (one2many): NEW relation: account_edi_proxy_client.user
 ---XML records in module 'account_edi_proxy_client'---
 NEW ir.model.access: account_edi_proxy_client.access_account_edi_proxy_user
-NEW ir.model.constraint: account_edi_proxy_client.constraint_account_edi_proxy_client_user_unique_edi_identification_per_format
-NEW ir.model.constraint: account_edi_proxy_client.constraint_account_edi_proxy_client_user_unique_id_client
```

Comment:
Same here. Why this removal ?
Othzrwise lgtm. Thanks

Reply:
> Same here. Why this removal ? Othzrwise lgtm. Thanks

@legalsylvain Thank you for reviewing it.
Maybe during the Perform Analysis it went wrong somewhere and I'm going through it to fix it. In `upgrade_analysis.txt` there is no note about constraints. I don't understand

Reply:
That's because you have an outdated version. The constraints were added some weeks ago.

Reply:
> That's because you have an outdated version. The constraints were added some weeks ago.

@pedrobaeza that seems to be the right reason. I will update the new code. Thank